examples/tutorials/advanced/panel_extension.py

```diff
@@ -0,0 +1,122 @@
+r"""
```

Comment:
I wonder if it's better to rename the file to "work_with_panel.py" or "working_with_panel.py".

The idea is that we may have more tutorials/examples to show how PyGMT works with other packages (e.g., Matplotlib, ObsPy, GeoPandas), so a consistent naming style is better.

Reply:
Yep, the file name can be definitely improved, this was just a quick first idea :sweat_smile:. I changed it to "working_with_panel.py" in commit https://github.com/GenericMappingTools/pygmt/pull/2498/commits/8f939320b243634d5ec792f267c96fe86f03d7f7.